devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -223,21 +223,26 @@ def reset(endpoint = 1) {
 }
 
 def childOnOff(deviceNetworkId, value) {
-	def switchId = deviceNetworkId?.split(":")[1] as Integer
+	def switchId = getSwitchId(deviceNetworkId)
 	sendHubCommand onOffCmd(value, switchId)
 }
 
 def childRefresh(deviceNetworkId) {
-	def switchId = deviceNetworkId?.split(":")[1] as Integer
+	def switchId = getSwitchId(deviceNetworkId)
 	sendHubCommand refresh(switchId)
 }
 
 def childReset(deviceNetworkId) {
-	def switchId = deviceNetworkId?.split(":")[1] as Integer
+	def switchId = getSwitchId(deviceNetworkId)
 	log.debug "Child reset switchId: ${switchId}"
 	sendHubCommand reset(switchId)
 }
 
+def getSwitchId(deviceNetworkId) {
+	def split = deviceNetworkId?.split(":")
+	return (split.length > 1) ? split[1] as Integer : 1
```

Comment:
otherwise 1?

Reply:
it's the default argument to all the functions

Reply:
and considering it's not working now it can't be much worse

Reply:
I rather fail out and not send any message

Reply:
fair enough, I'll fix it